libs/dal/src/repositories/message-template/types.ts

```diff
@@ -1,17 +1,7 @@
-import { EmailBlockTypeEnum, TemplateVariableTypeEnum, TextAlignEnum } from '@novu/shared';
-
-export interface IEmailBlock {
-  type: EmailBlockTypeEnum;
-  content: string;
-  url?: string;
-  styles?: {
-    textAlign?: TextAlignEnum;
-  };
-}
-
-export interface ITemplateVariable {
-  type: TemplateVariableTypeEnum;
-  name: string;
-  required: boolean;
-  defaultValue?: string | boolean;
-}
```

Comment:
This types are used by different entities so I moved them out to the types folder for easier sharing and importing, so there weren't dependencies of the message template entity around.